qbft/types.go

```diff
@@ -27,6 +27,10 @@ type Syncer interface {
 	SyncDecidedByRange(identifier types.MessageID, from, to Height)
 }
 
+type RootGetter interface {
+	GetRoot() ([]byte, error)
```

Comment:
IMO this should be in [qbft/spectest/comparable/types.go](https://github.com/bloxapp/ssv-spec/pull/192/files#diff-6a541211891c5077274df95b7f8e284153b952f31559ff0d72ae38d86d5f56c0) sine it is used only there

Reply:
general types is not a best practice in go, but it is sometimes necessary, for types that are re-used across the codebase. this is not the case here.

Reply:
I have more of a java background, but the internet (how one learns to code these days) tells me that it is a fine practice in golang.
Also there are types in `qbft` and `ssv` package that implement that interface.

Reply:
yeah I saw this is handled in another PR so resolving, though this is a good practice of software engineering in general and not just go, it's better for readability and easier to understand when you define stuff next to where you use them.